components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2245,6 +2252,18 @@ private void parseHashAlgorithm(OMElement oauthConfigElem) {
         }
     }
 
+    private void parseEnableHashMode(OMElement oauthConfigElem) {
+
+        OMElement hashModeElement = oauthConfigElem
+                .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.ENABLE_HASH_MODE));
```

Comment:
This needs to be added to identity.xml and should be documented

Reply:
Added the configuration in this PR [1].
[1] https://github.com/wso2/carbon-identity-framework/pull/1542

Reply:
Done. 
Validated the hash persistence processor